causalml/inference/tree/utils.py

```diff
@@ -0,0 +1,228 @@
+"""
```

Comment:
Since these are related to feature engineering/transformation, let's move it to `causalml/features.py`.